tests/unit/openstack/heat/test_template.py

```diff
@@ -3,6 +3,24 @@
 from ignition.utils.propvaluemap import PropValueMap
 
 class TestHeatInputUtil(unittest.TestCase):
+    def setUp(self):
```

Comment:
Can't this variables be declared within the test case as these are not being used in any other tests?

Reply:
I have used this variable in following function

```
def test_filter_password_from_dictionary(self):
        result = self.obj.filter_password_from_dictionary(self.template)
        self.assertEqual(result, self.filtered_template) 
```
And also used in other test cases